server/server.js

```diff
@@ -1,19 +1,24 @@
 const { ApolloServer } = require('apollo-server-express');
 const express = require('express');
+const jwt = require('jsonwebtoken');
 const { typeDefs, resolvers } = require('./schemas');
 const db = require('./config/connection');
+const { authMiddleware } = require('./utils/auth');
+const dotenv = require('dotenv');
```

Comment:
You don't need to create a variable when dealing with dotenv. It should be the following:
`require('dotenv').config`
You may want to have this as first line of the file for formatting perspective as well